allen-cell-animated/simularium-website
# 100% test coverage of state selectors + some cleanup

Problem
=======
We want more test coverage. A significant step towards completing #182 

Solution
========
* Add more tests to state selectors to get 100% test coverage.

    Work still remaining to close out this issue: test coverage for container-specific selector files. For example, src/containers/ModelPanel/selectors.ts has no tests. Because the test file doesn't exist, Jest currently doesn't calculate code coverage for that container's selectors. Ironic!

* I also renamed `visibleAgentKeys` in `state.selection` to `agentVisibilityMap` and `highlightedAgentKeys` to `agentHighlightMap` to make them more accurate and less confusing, per discussion with @meganrm 

* Removed some old unused selectors

## Type of change

* This change requires updated or new tests

Steps to Verify:
----------------
1. Check that test coverage is 100% for all selectors under src/state - this should be taken care of by GitHub Actions check
2. Check that variable renaming didn't break anything - `npm start`

Keyfiles (delete if not relevant):
-----------------------
1. src/state/selection/selectors/selectors.test.ts
2. src/state/trajectory/selectors/selectors.test.ts

Everything else is variable renaming or deleting of unused selectors.

Thanks for contributing!
